pkgs/build-support/cc-wrapper/default.nix

```diff
@@ -629,6 +629,12 @@ stdenvNoCC.mkDerivation {
       echo " -L${libcxx_solib}" >> $out/nix-support/cc-ldflags
     ''
 
+    ## Prevent the compiler from seeing /usr/include. There is a desire to
+    # achieve this through alternate means. See discussion in NixOS/nixpkgs#191152.
```

Comment:
The Darwin‐related content from the comment above the conditional in the other file should move here (just the “on macOS … hopefully shouldn’t matter” bit should be fine).

Reply:
Moved. Good catch, thanks!

Reply:
Still calls it a patch, but otherwise SGTM.

Reply:
s/skipping the patch/dropping the flag/